docs/migration/3_0.md

```diff
@@ -77,6 +77,13 @@ The property `childRoutes` of config `cmsComponents` changed type from `Route[]`
 
 - ConverterService constructor is now using `UnifiedInjector` instead of standard `Injector` 
 
+### Property renamed in `SearchConfig` interface
+
+| Old Name | New Name 
+| --- | --- 
+| `sortCode` | `sort` 
+
+
```

Comment:
is it also worth mentioning that the two interfaces (ProductListRouteParams and SearchCriteria) from the `product-list-component.service` that was removed and moved to `product-list.model`?

Reply:
Good question.  I thought we needed to write schematics and if not possible, write some doc.

Reply:
I don't think it would be in any schematics but maybe in the docs would help

Reply:
just going to put it here, the branch is called `featrue/xxx` and not `feature/xxx`, but it doesn't seem to be a problem (just pointing it out)

Reply:
Oops, yup, we did a typo on the branch name.

Reply:
> is it also worth mentioning that the two interfaces (ProductListRouteParams and SearchCriteria) from the `product-list-component.service` was removed and moved to `product-list.model`?

For these, we looked back and those 2 interfaces were not exported before they were moved.  Therefore, not a breaking change.

Reply:
🟢